samples/LiveEffect/src/main/cpp/CMakeLists.txt

```diff
@@ -42,5 +42,5 @@ target_link_options(liveEffect PRIVATE "-Wl,-z,max-page-size=16384")
 
 # Enable optimization flags: if having problems with source level debugging,
 # disable -Ofast ( and debug ), re-enable it after done debugging.
-target_compile_options(liveEffect PRIVATE -Wall -Werror "$<$<CONFIG:RELEASE>:-Ofast>")
+target_compile_options(liveEffect PRIVATE -Wall -Werror "$<$<CONFIG:RELEASE>:-O3 -ffast-math>")
```

Comment:
This may be a great change. But it seems unrelated to the new PowerPlay app.  If you see room for improvement, it would be easier to submit those as separate PRs.

Reply:
No issues!